commands/build/lib/build.js

```diff
@@ -54,6 +55,7 @@ const config = ({
 
   const auth = typeof author === 'object' ? `${author.name} <${author.email}>` : author || '';
   const moduleName = name.split('/').reverse()[0];
+  const extensions = ['.mjs', '.js', '.jsx', '.json', '.node', '.tsx', '.ts'];
```

Comment:
Only add .ts and .tsx when typescript is enabled?

Reply:
Yes that is a good idea